rootfs/filesystem/etc/udhcpc/script.sh

```
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+case "$1" in
+  bound|renew)
+    ip addr flush dev "$interface"
```

Shellcheck (0.11.0)

[warning] interface is referenced but not assigned.

(SC2154)